File: Difference-Between-Element-Sum-and-Digit-Sum-of-an-Array.py
class Solution(object):
    def differenceOfSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        sum1 = 0
        sum2 = 0

        for i in range(len(nums)):
            sum1 += nums[i]
            n = nums[i]
            while n > 0:
                sum2 += n % 10
                n //= 10

        return abs(sum1 - sum2)

File: Difference-Between-Element-Sum-and-Digit-Sum-of-an-Array.java
class Solution {
public int differenceOfSum(int[] nums) {
    int sum1 = 0, sum2 = 0;

    for (int i = 0; i < nums.length; i++) {
        sum1 += nums[i];
        int n = nums[i];
        while (n > 0) {
            sum2 += n % 10;
            n /= 10;
        }
    }

    return Math.abs(sum1 - sum2);
}

}